packages/dds/sequence/src/sequence.ts

```diff
@@ -299,7 +299,7 @@ export abstract class SharedSegmentSequence<T extends ISegment>
 
 		this.client.prependListener("delta", (opArgs, deltaArgs) => {
 			const event = new SequenceDeltaEvent(opArgs, deltaArgs, this.client);
-			if (opArgs.stashed !== true && event.isLocal) {
+			if (event.isLocal) {
```

Comment:
this is where sequence submits its ops based on events, now we submit all local ops.